src/utils/useWaitForDOMRef.js

```diff
@@ -2,6 +2,7 @@ import ownerDocument from 'dom-helpers/ownerDocument';
 import { useState, useEffect } from 'react';
 
 const resolveRef = ref => {
+  if(typeof document === 'undefined') return undefined;
```

Comment:
So I'm pretty sure returning `undefined` here isn't the correct behavior for this use case, does anyone have a better suggestion for what to return here?